frontend/apps/filemanager/filemanagerfilesearcher.lua

```diff
@@ -111,30 +101,23 @@ function FileSearcher:onShowFileSearch()
                     end,
                 },
                 {
-                    text = _("Current folder"),
-                    enabled = true,
+                    text = _("Home folder"),
+                    enabled = G_reader_settings:has("home_dir"),
                     callback = function()
-                        self.path = self.ui.file_chooser and self.ui.file_chooser.path or self.ui:getLastDirFile()
                         self.search_value = self.search_dialog:getInputText()
-                        if self.search_value == dummy then -- probably DELETE this if/else block
-                            self.use_previous_search_results = true
-                        else
-                            self.use_previous_search_results = false
-                        end
```

Comment:
Are you sure about this?

Reply:
`use_previous_search_results` is not used anywhere in the repository.
Maybe somebody had a plan to develop something?

Reply:
I guess that someone would be me. >_> Anyway, looks like you're right.